qemux86-64.robot

```diff
@@ -12,6 +12,8 @@ Preparing test environment
   Set Suite Variable    ${image}    %{image}
   File Should Exist     ${image}  msg="Provided images file does not exist"
   Set Suite Variable    ${application_repo}    https://github.com/resin-io-projects/alpine-barebone.git
+  Set Suite Variable    ${variable_name}    name
```

Comment:
We can drop these as suite variables and just check by running the tests with a specific variable name and value - Commented below